Resources/Prototypes/_LostParadise/Datasets/dreamadverbs.yml

```
@@ -0,0 +1,139 @@
+﻿- type: dataset
+  id: dreamadverbs
+  values:
+    #ss13
+  - случайно
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)